aixplain/modules/finetune/hyperparameters.py

```diff
@@ -56,6 +65,12 @@ def __post_init__(self):
         if self.epochs > EPOCHS_MAX_VALUE:
             raise ValueError(f"epochs must be one less than {EPOCHS_MAX_VALUE}")
```

Comment:
less or equal

Reply:
In this case, we should only raise ValueError if the number of epochs is greater than the maximum allowed value (`EPOCHS_MAX_VALUE`), so the logic is correct.

Reply:
I'll change the message to ensure it's correct.